<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingUI</class>
 <widget class="QWidget" name="SettingUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1572</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,9">
   <item>
    <widget class="QListWidget" name="listWidget_settings"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_changePassword">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>更换密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_saveSettings">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>保存设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_readSettings">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>读取设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_loginSettings">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>登陆设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
